Sevenpros/Teamwork
# 168972089 Build out Signup Endpoint using Database

#### What does this PR do?
Having Singup endpoint working using database
#### Description of Task to be completed?
Have the following features:
* **POST/ api/v2/auth/signup** should be working 
* Authentication function should not crush the server in case of invalid token
* create directory to contain all database related files
* having one function to connect to db inside the model index file
* Creating Function to insert new user
* creating function to get one specific user
* check if user already has an account.
#### How should this be manually tested?
* After cloning this project navigate to the project folder 
You can use post man to check if this endpoint is working 
* You can do the request for getting all articles using **(GET/ api/v1/feeds)** with invalid token passed into header to test if the authentication is properly handling the error. (token is provided once a user successfully logged in)'.
### user successfully signed up
![image](https://user-images.githubusercontent.com/26649756/66414969-cc49df80-e9fa-11e9-8f2d-32b67b4a83cd.png)
#### When user already exist !!
![image](https://user-images.githubusercontent.com/26649756/66415072-fac7ba80-e9fa-11e9-925a-8cb6a665f3c0.png)

### Invalid Token
![image](https://user-images.githubusercontent.com/26649756/66415184-5134f900-e9fb-11e9-8d04-bd4819112df8.png)


#### What are the relevant pivotal tracker stories?
[ #168972089](https://www.pivotaltracker.com/story/show/168972089);
